### ISSUE TITLE
A problem occurred configuring project ':material'

### ISSUE THREAD

**Issue by karattejoseph** [afollestad/material-dialogs#661]
After installing material-dialogs, apps works fine. But When I try to generate the signed apk, studio throwing the following error.

Error:A problem occurred configuring project ':app'.
> A problem occurred configuring project ':material'.
  > Cannot access first() element from an empty List

Any help appreciated

**Comment by afollestad** [comment]
This is related to a dependency that's used, I'm waiting on a new release.

**Comment by afollestad** [comment]
Also a duplicate of https://github.com/afollestad/material-dialogs/issues/614#issuecomment-135624964
